### PR TITLE
Remove docs rendering from duckduckhack root module

### PR DESCRIPTION
**Pull request by jbarrett**
Depends on : https://github.com/duckduckgo/community-platform/pull/1098

This removes the documentation build completely from publisher.

### CODE REVIEW

**Comment by jbarrett** [comment]
For those who were following along in https://github.com/duckduckgo/duckduckgo-publisher/pull/212, page build has gone from ~1s to ~0.1s

**Comment by nilnilnil** [comment]
Why does it need to be here at all? If it does, please add a comment as to why the stub is required.

File: lib/DDG/Publisher/Site/Duckduckhack/Root.pm [hunk 1]
```diff
@@ -2,344 +2,13 @@ package DDG::Publisher::Site::Duckduckhack::Root;
...
-my $dir_output = '/usr/local/ddg/www-static/duckduckhack.com/tmp';
-
-sub pages {
-	return {} if $ENV{DDH_SKIP_DOCS_BUILD};
-	my $self = shift;
-
-	my %pages = ();
-
-	# Old index.
-	$pages{'index'} = sub {
```
Comment (moollaza) [inline lib/DDG/Publisher/Site/Duckduckhack/Root.pm]: @jbarrett this stuff we still need I beleive -- it's really silly, but this file contains the code to build duckduckhack.com **and** the docs.

So `duckduckhack` should now only refer to the static page -- unless we're not handling that in publisher anymore either?

Comment (jbarrett) [inline lib/DDG/Publisher/Site/Duckduckhack/Root.pm]: Returning an empty hashref for pages leaves the DDH index intact.

**Comment by jbarrett** [comment]
@nilnilnil I would, but I don't know why it needs to be there.

Do we need comments for all of these bare modules?

https://github.com/duckduckgo/duckduckgo-publisher/blob/master/lib/DDG/Publisher/Site/Donttrackus/Root.pm

https://github.com/duckduckgo/duckduckgo-publisher/blob/master/lib/DDG/Publisher/Site/Dontbubbleus/Root.pm

**Comment by jbarrett** [comment]
So this should be good to go ([docs](https://duck.co/duckduckhack/ddh-intro) are now being updated and built directly on duck.co).

We still need the bare module for DDH front page development, so that is left intact. Thanks!

**Comment by moollaza** [comment]
This LGTM :+1:

**Pull request merged**